R/animint.R

```diff
@@ -94,7 +94,9 @@ parsePlot <- function(meta){
       ##    theme.pars$panel.grid then from theme.pars$line
       for(i in 1:length(pars)) {
         if(is.null(pars[[i]])) pars[[i]] <- 
-          if(!is.null(theme.pars$panel.grid[[i]])) {
+          if("element_blank" %in% class(theme.pars$panel.grid)) {
+            theme.pars$line[[i]]
+          } else if( !is.null(theme.pars$panel.grid[[i]]) ) {
```

Comment:
I think this is a better solution than my previous attempt.  While this generates an error
```
x <- list()
x[[1]]
Error in x[[1]] : subscript out of bounds
```
this returns `NULL` like I want
```
x[["colour"]]
NULL
```